### PR TITLE
June 2023's blog post for Liberty Tools and Liberty build plugins

### LINKED ISSUE #3184: Blog post: Updates for Liberty Tools and Liberty build plugins
A blog post for the latest releases of Liberty Tools and Liberty build plugins.

### PR DESCRIPTION
**Pull request by yeekangc**
New blog post for Liberty Tools and Liberty build plugins.

This is for #3184.

Draft: https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2023/07/10/liberty-ide-tools-maven-gradle-plugins-for-java-developers.html

### CODE REVIEW

**Comment by yeekangc** [comment]
@scottkurz @cherylking, can you review and approve the technical content?

**Comment by yeekangc** [comment]
@GraceJansen @lauracowen, can one of you please review and approve for inclusion from a planning standpoint?

**Comment by yeekangc** [comment]
@dmuelle, we can publish the week of July 10 when you and team are ready.  Thank you.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (lauracowen) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]: I would delete all these commented lines, just for tidiness as they're not needed any more.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (lauracowen) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
Please share your feedback and what you would like to see in future releases via issues in the GitHub repositories for link:https://github.com/OpenLiberty/liberty-tools-eclipse/issues/new[Liberty Tools for Eclipse IDE], link:https://github.com/OpenLiberty/liberty-tools-vscode/issues/new[Liberty Tools for Visual Studio Code], and link:https://github.com/OpenLiberty/liberty-tools-intellij/issues/new[Liberty Tools for IntelliJ IDEA].
```

Just seems a little more straightforward than parsing a negative.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (lauracowen) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
Liberty build plugins enable developers to easily build and package their applications to run on Liberty using the popular Java build tools, Maven and Gradle. It's always recommended to use the latest versions of these build plugins to ensure you have the most recent fixes and enhancements, Liberty Maven Plugin 3.8.2 and Liberty Gradle Plugin 3.6.2. For example, starting with the 3.6 release, the Liberty Gradle Plugin supports running with Gradle 8.x. For more information, see link:https://github.com/OpenLiberty/ci.maven/releases[Liberty Maven Plugin release notes] and link:https://github.com/OpenLiberty/ci.gradle/releases[Liberty Gradle Plugin release notes].
```

It's easier to parse the version number next to its respective name, rather than having to mentally map the versions to the names, especially for someone skimming the info.

Comment (lauracowen) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]: I'm not sure what the "For example" sentence is referring to. What's it an example of? It feels more like an explanation of what version of Gradle you need, and that the equivalent info about Maven is therefore missing.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (lauracowen) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
To use Liberty Maven Plugin 3.8.2, specify the following dependency in your Maven build file:
```
Not sure if "dependency" is the right word here but it needs something after "the following", which isn't a noun in its own right.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (lauracowen) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]: Whatever you do for Maven above after "the following..." do here too.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (lauracowen) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
If you have requests or run into problems, create new issues in the GitHub repositories for link:https://github.com/OpenLiberty/ci.maven/issues/new[Liberty Maven Plugin] and link:https://github.com/OpenLiberty/ci.gradle/issues/new[Liberty Gradle Plugin].
```

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (lauracowen) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
Like the Open Liberty runtime itself, all of Liberty Tools, Liberty Maven Plugin, and Liberty Gradle Plugin are open source and are developed under the link:https://github.com/OpenLiberty[OpenLiberty organization on GitHub]. You are welcome to contribute directly and make Liberty even better!
```

**Review by lauracowen — CHANGES_REQUESTED** [review]
Some minor suggestions but otherwise looks good to me. I'll leave it to @GraceJansen to schedule/plan though.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (cherylking) [reply posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]: It's an example of why you would want to use a recent version of the plugin, because of added support for Gradle 8.x.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (cherylking) [reply posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]: Maybe `configuration` instead of `dependency`.

**Review by cherylking — DISMISSED** [review]
approving technical content

**Review by scottkurz — DISMISSED** [review]
LGTM

**Review by gljrobinson — DISMISSED** [review]
LGTM.

**Comment by gljrobinson** [comment]
Happy to publish the week beginning the 10th July if @dmuelle has capacity after getting back from vacation.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (yeekangc) [reply posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]: Will change it to "For example, in Liberty Gradle Plugin 3.6, we added support for Gradle 8.x.”

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (dmuelle) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
seo-description: New releases of Liberty Tools, Liberty Maven Plugin, and Liberty Gradle Plugin help Java developers easily write applications in Eclipse IDE, Visual Studio Code, and IntelliJ IDEA and build them using Maven and Gradle.
```

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (dmuelle) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
blog_description: "New releases of Liberty Tools, Liberty Maven Plugin, and Liberty Gradle Plugin help Java developers easily write applications in Eclipse IDE, Visual Studio Code, and IntelliJ IDEA and build them using Maven and Gradle."
```

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (dmuelle) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
Liberty Tools 23.0.6 are now available for Java developers to write cloud-native applications with Liberty.
```

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (dmuelle) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
Liberty Tools support popular integrated development environments (IDEs) such as Eclipse IDE, Visual Studio Code, and IntelliJ IDEA, allowing developers to focus on their code and rapidly iterate by using Liberty's link:/docs/latest/development-mode.html[dev mode] in their preferred IDE. Moreover, Liberty Tools provide editing assistance for developers to easily make changes to the Liberty configuration files, and coding assistance for writing applications that use Jakarta EE (9.x and later) and MicroProfile (3.x and later) APIs.
```

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (dmuelle) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
Please share your feedback and what you would like to see in future releases by opening issues in the GitHub repositories for link:https://github.com/OpenLiberty/liberty-tools-eclipse/issues/new[Liberty Tools for Eclipse IDE], link:https://github.com/OpenLiberty/liberty-tools-vscode/issues/new[Liberty Tools for Visual Studio Code], and link:https://github.com/OpenLiberty/liberty-tools-intellij/issues/new[Liberty Tools for IntelliJ IDEA].
```

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (dmuelle) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
== Easily build, package, and run applications with Liberty Maven & Gradle plugins
```

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc
Comment (dmuelle) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
Like the Open Liberty runtime itself, Liberty Tools, Liberty Maven Plugin, and Liberty Gradle Plugin are open source and are developed under the link:https://github.com/OpenLiberty[OpenLiberty organization on GitHub]. You are welcome to contribute directly and make Liberty even better!
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
a few minor ID suggestions. Also, would you change the filename to target tomorrow (7/11) as the publish date?

**Review by dmuelle — COMMENTED** [review]
LGTM, thanks

**Review by dmuelle — DISMISSED** [review]

**Comment by dmuelle** [comment]
@yeekangc @lauracowen - looks like this is blocked by Laura's change request, though I believe YK has made all the changes. Since Laura won't be in until tomorrow, we should change the date to reflect the new publish date. But maybe wait until she has a chance to review/approve in case anything else comes up.

**Comment by yeekangc** [comment]
Updated filename to reflect the new publish date.

**Review by lauracowen — APPROVED** [review]

**Pull request merged**
